rc/lsp.kak

```diff
@@ -961,8 +975,8 @@ define-command lsp-enable-window -docstring "Default integration with kak-lsp in
     lsp-inline-diagnostics-enable window
     lsp-diagnostic-lines-enable window
 
-    map window goto d '<esc>: lsp-definition<ret>' -docstring 'definition'
-    map window goto r '<esc>: lsp-references<ret>' -docstring 'references'
+    map window goto d '<esc>: lsp-definition<ret>'
```

Comment:
Any specific reason to remove docstring here and below?

Reply:
Oi!